nginx.conf

```diff
@@ -13,25 +13,29 @@ http {
     # include C:/nginx/conf/mime.types;
 
     server {
-        root dist/; # necessary to avoid nginx injecting root into try_files directives
         
-        # windows,  Paths in a configuration file must be specified in UNIX-style using forward slashes.
+        # windows, Paths in a configuration file must be specified in UNIX-style using forward slashes.
         # root <ABSOLUTE PATH TO dist directory content>;
+        root dist/; # necessary to avoid nginx injecting root into try_files directives
 
         listen 8080;
         server_name localhost;
         server_name 127.0.0.1;
-        sendfile off;
 
+        # Enable static gzip
+        gzip_static on;
+        
+        # Disable dynamic compression (optional, and not recommended if you're proxying)
+        gzip off;
+        gzip_types text/css text/javascript application/javascript application/x-javascript;
+
+        sendfile on;
+        
         location / {
             try_files $uri$args $uri$args/ /index.html;
         }
 
-        # serve static content
-        location /static/ {
-            # path for static files
-            root .;
-        }
+        # remove this?!
```

Comment:
I think we should remove this or find a way for dynamic proxy?
As we can have call to 0.0.0.0:1995 via https or as per configured port